types/tslint.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "dtslint/dtslint.json",
+  "rules": {
+    "no-redundant-jsdoc": false,
+    "semicolon": false,
+    "unified-signatures": true,
+    "whitespace": false,
+    "strict-export-declare-modifiers": false
```

Comment:
After the recent changes, I think only `semicolon` and `whitespace` need to be disabled?

Reply:
It doesn't look like enabling [`strict-export-declare-modifiers`](https://github.com/microsoft/dtslint/blob/master/docs/strict-export-declare-modifiers.md) will work? 

```
ERROR: 6:19   strict-export-declare-modifiers  All declarations in this module are exported automatically. Prefer to explicitly write 'export' for clarity. If you have a good reason not to export this declaration, add 'export {}' to the module to shut off automatic exporting. See: https://github.com/Microsoft/dtslint/blob/master/docs/strict-export-declare-modifiers.md
```

Reply:
:thinking: interesting, I tried

with tslint config

```json
{
  "extends": "dtslint/dtslint.json",
  "rules": {
    "semicolon": false,
    "whitespace": false
  }
}
```

and running

```bash
$ node --version
v13.6.0

$ npm --version
6.13.4

$ npx dtslint --version
2.0.5

$ git rev-parse HEAD
bfe4f07053a2f8e3c2d6b34eb1417f6da9cd3064

$ npm run test-types

> remark-toc@6.0.0 test-types /home/developer/Projects/OpenSource/remark-toc
> dtslint types

```

I don't see that error :thinking:

Reply:
I had to remove my `node_modules` to see it passing, but now CI is failing 😭